CMakeLists.txt

```diff
@@ -154,19 +156,15 @@ endif()
 if (SvtEnc_FOUND AND NOT WITH_SvtEnc_PLUGIN)
     list(APPEND REQUIRES_PRIVATE "SvtAv1Enc")
 endif()
-if ((JPEG_DECODER_FOUND AND NOT WITH_JPEG_DECODER_PLUGIN) OR (JPEG_ENCODER_FOUND AND NOT WITH_JPEG_ENCODER_PLUGIN))
+if (JPEG_FOUND AND NOT (WITH_JPEG_DECODER_PLUGIN AND WITH_JPEG_ENCODER_PLUGIN))
     list(APPEND REQUIRES_PRIVATE "libjpeg")
 endif()
-# TODO: JPEG
-if (OpenJPEG_FOUND AND NOT WITH_OpenJPEG_PLUGIN)
+if (OpenJPEG_FOUND AND NOT (WITH_OpenJPEG_DECODER_PLUGIN AND WITH_OpenJPEG_ENCODER_PLUGIN))
     list(APPEND REQUIRES_PRIVATE "libopenjp2")
 endif()
 if (LIBSHARPYUV_FOUND)
     list(APPEND REQUIRES_PRIVATE "libsharpyuv")
 endif()
-if (OpenJPEG_FOUND AND NOT WITH_OpenJPEG_PLUGIN)
-    list(APPEND REQUIRES_PRIVATE "OpenJPEG")
-endif()
```

Comment:
@farindk This is the minimal snippet that is obviously incorrect and really needs to go in before the next release, the rest can wait to be fleshed out more...